lib/active_merchant/billing/gateways/rapyd.rb

```diff
@@ -294,10 +299,17 @@ def parse(body)
         JSON.parse(body)
       end
 
+      def url(action, mode = :default)
+        if mode == 'payment_redirect' && action == 'payments'
```

Comment:
The default argument is using the symbol `:` but your conditional checks against a string. Will the conditional still succeed if the value is `:payment_redirect`?

Reply:
Hi @naashton thanks good point, I've added an update if for some reason the mode value is a symbol.